lib/auth/desktop.go

```diff
@@ -70,6 +73,12 @@ func (s *Server) GenerateWindowsDesktopCert(ctx context.Context, req *proto.Wind
 		// CRL is required for Windows smartcard certs.
 		CRLDistributionPoints: []string{req.CRLEndpoint},
 	}
+	if s.license != nil {
+		certReq.ExtraExtensions = append(certReq.ExtraExtensions, pkix.Extension{
+			Id:    LicenseOID,
+			Value: []byte("enterprise"),
```

Comment:
Can you use `modules.GetModules().BuildType` here?

Reply:
Additionally, instead of putting this in the cert request, can the auth server do it automatically?

Reply:
I can, I've changed it.
It _is_ auth server doing the job - the req modified here is the final piece that we pass almost directly to `x509.CreateCertificate`